_rules/sequentially-focusable-element-has-visible-focus-oj04fd.md

```diff
@@ -0,0 +1,232 @@
+---
+id: oj04fd
+name: Element in sequential focus order has visible focus
+rule_type: atomic
+description: |
+  This rule checks that each element in sequential focus order has some visible focus indication.
+accessibility_requirements:
+  wcag20:2.4.7: # Focus Visible (A)
```

Comment:
🤔 
Actually, 2.4.7 is only level AA in WCAG 2.1 but will move to level A in WCAG 2.2.

Not sure how to handle that…